examples/advanced/gnn-pt/README.md

```diff
@@ -0,0 +1,38 @@
+# GraphSAGE with PyTorch and PyG
+
+Example of using [NVIDIA FLARE](https://nvflare.readthedocs.io/en/main/index.html) to train an image classifier
+using federated averaging ([FedAvg](https://arxiv.org/abs/1602.05629))
+and [PyTorch](https://pytorch.org/) as the deep learning training framework.
+
+> **_NOTE:_** This example uses the [PPI]([http://snap.stanford.edu/graphsage/#code]) dataset and will load its data within the trainer code.
+
+You can follow the https://github.com/pyg-team/pytorch_geometric/blob/master/examples/graph_sage_unsup_ppi.py for graphsage code.
+
+
+### 1. Install NVIDIA FLARE
+
+Follow the [Installation](https://nvflare.readthedocs.io/en/main/quickstart.html) instructions.
+Install additional requirements:
+
+```
+pip3 install -r requirements.txt
+```
+
+### 2. Run the experiment
+
+Use nvflare simulator to run the hello-examples:
+
+```
+nvflare simulator -w /tmp/nvflare/ -n 2 -t 2 gnn-pt/jobs/gnn-pt
+```
+
+### 3. Access the logs and results
+
+You can find the running logs and results inside the simulator's workspace/simulate_job
+
+```bash
+$ ls /tmp/nvflare/simulate_job/
+app_server  app_site-1  app_site-2  log.txt
+
+```
+
```

Comment:
Can you add some expected results and training curves here like in other examples, e.g. this [one](https://github.com/NVIDIA/NVFlare/blob/dev/examples/advanced/cifar10/cifar10-sim/README.md#32-centralized-training)?